src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -442,7 +442,7 @@ doCommit = use bsMode >>= \mm -> case mm of
     Just m -> do
         txrs <- if m == Transactional
           then do
-              modify' (over bsTxId succ)
+              bsTxId %= succ
```

Comment:
Can we revert this change?  It's not obvious how this impacts strictness.  And it's unrelated to the rest of the PR.